resources/packs/gcp/gcp.lr

```diff
@@ -867,6 +869,26 @@ private gcp.project.bigqueryService.dataset @defaults("id"){
   routines() []gcp.project.bigqueryService.routine
 }
 
+// GCP BigQuery dataset access entry
+private gcp.project.bigqueryService.dataset.accessEntry {
+  // Internal ID
+  id string
+  // Dataset ID
+  datasetId string
+  // Role of the entity
+  role string
+  // Type of the entity
+  entityType string
+  // Entity (individual or group) granted access
+  entity string
+  // View granted access (entityType must be ViewEntity)
+  viewRef dict
```

Comment:
i believe we don't need to retrieve the actual view/routine/dataset for writing our policies. If in the future we need the actual reference we can add `view() gcp.project.bigqueryService.view`, for example. For now I think it's not needed